source/scpd/Cpp.d

```diff
@@ -76,40 +79,39 @@ private nothrow @nogc extern(C++) void cpp_unordered_map_assign (K, V)(
 private pure nothrow @nogc @safe extern(C++) size_t cpp_unordered_map_length (K, V)(
     const(void)* map);
 
-// @bug with substitution
-// https://issues.dlang.org/show_bug.cgi?id=20679
-//private pure nothrow @nogc @safe extern(C++) unordered_map!(K, V)* cpp_unordered_map_create (K, V)();
-
-/// Create a new unordered map with the given Key/T types, and return a pointer to it
-private pure nothrow @nogc @safe extern(C++) void* cpp_unordered_map_create (K, V)();
-
 /// Rudimentary bindings for std::unordered_map
 extern(C++, (StdNamespace))
 public struct unordered_map (Key, T, Hash = hash!Key, KeyEqual = equal_to!Key, Allocator = allocator!(pair!(const Key, T)))
 {
-    private void* ptr;
+    private ulong[56 / ulong.sizeof] _data;
```

Comment:
How'd you get this number?

Reply:
It's a magic number, verified here

https://github.com/bosagora/agora/pull/2066/files#diff-2fff5f741967021f44722f88e37effa07123b287d13e843ff246b43d91a7d8d4R478

Reply:
It will probably be different for Mac OSX tho, CI will tell

Reply:
Yes I'm having similar problems with https://github.com/bosagora/agora/pull/2065. There's a size mismatch on one platform (linux) but not another (macos). So much for C++ portability huh? :o)